### DISCUSSION TITLE
Support image preview in Warp

### DISCUSSION THREAD

**Discussion by spencercjh** [sxyazi/yazi#539] [1. Q&A]
https://www.warp.dev/

warp
v0.2024.01.16.16.31.stable_01

**Comment by spencercjh** [comment]
![image](https://github.com/sxyazi/yazi/assets/29922079/103c32a1-1159-4232-a5b3-9abc3f26bb45)

**Comment by sxyazi** [comment]
Please create a feature request instead of discussing, https://github.com/sxyazi/yazi/issues/new?template=feature.yml

**Comment by sxyazi** [comment]
Unfortunately, Warp currently does not support any graphical protocols, https://github.com/warpdotdev/Warp/issues/26, and until they implement it, Yazi cannot display images in Warp :(

**Reply by spencercjh** [reply]
I'm completely ignorant in this area, thanks for the introduction.
